Feature/create title

jonastore opened the pull request.

jonastore committed:
first commit

jonastore committed:
add support to create multiple questions in questionnaire

jonastore committed:
add possibility to create several questions from the create-questionnaire view

jonastore committed:
add ActionTypes for single question

jonastore committed:
add functionality for create and fetch single questions

jonastore committed:
add mapStateToProps function

jonastore committed:
add support for creations and fetching single questions

jonastore committed:
create createQuestion container

jonastore committed:
add ListableQuestion component

izabelrosen commented:
Please change merge to dev instead of master!

SebastianGerS reviewed (commented):
The styling will be fixed later but I found a bug that have to be adressed. It apeard when whe added the "done" button, it covers the add button when creating questions and makes it impossible to add questions to the questionnaire

but asside from that everything seams to work well!

Commit:
bugfix for Add button in CreateQuestion

	* change possition of Add button in CreateQuestion component
	* add margin to some elements

	This fixes the bug where the Done button in the CreateQuesitonnaire component was covering the Add button in the CreateQuestion rendering the user unable to create questions
